charts/mapgl-js-api/values.yaml

```
@@ -61,7 +61,7 @@ env:
   MAPGL_IMMERSIVE_TILESET: web_immersive
   MAPGL_TRAFFICSERVER: https://traffic-proxy.ingress.host
   MAPGL_FLOORSSERVER: https://floors-api.ingress.host
```

Не совсем по ПР, но приведенный пример не работает. Только вот так, доменом
```suggestion
  MAPGL_FLOORSSERVER: floors-api.ingress.host
```
Можете поправить чтобы это было URL, а не домен? Этого же касается `MAPGL_TRAFFICSERVER`

---

Т.е. поясню, если воспрользоваться примером, то [тут](https://github.com/2gis/on-premise-helm-charts/pull/420/files#diff-5812937e73fcb4b5f12ef37d4b0881e7975b9d0550d05ef6b5079962873cbd4eR56) допишится еще один `https://`

---

Привет, эту штуку я поправил. 

Только по MAPGL_TRAFFICSERVER кажется все работать должно, я уточню, может в движке где-то у нас проблема.

---

@v-a-v Проверил еще раз, пробки норм работают, если задавать их как `MAPGL_TRAFFICSERVER: https://jam.api.2gis.com/`

---

@itanka9 можете разрезолвить конфликт?